src/Pages/Detail/Components/CreatorMessage.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from "react";
+import Styled from "styled-components";
+import axios from "axios";
+import SwiperCore, { Pagination } from "swiper";
+import { Swiper, SwiperSlide } from "swiper/react";
+import "swiper/swiper.scss";
+import "swiper/components/pagination/pagination.scss";
+import { API } from "./Utils";
+
+SwiperCore.use([Pagination]);
+
+const CreatorMessage = () => {
+  const [notice, setNotice] = useState([]);
+
+  useEffect(() => {
+    axios.get(API).then((res) => {
+      setNotice(res.data.notice);
+    });
+  }, []);
+  return (
+    <CreatorWrap>
+      <Swiper
+        spaceBetween={50}
+        slidesPerView={1}
+        pagination={{ clickable: true }}
+        scrollbar={{ draggable: true }}
+        // onSlideChange={() => console.log("slide change")}
+        // onSwiper={(swiper) => console.log(swiper)}
+      >
+        {notice.map(({ profile_image, nickname, date, description }) => (
+          <SwiperSlide>
+            <MessageItem img={profile_image} nickname={nickname} date={date}>
+              {description}
+            </MessageItem>
+          </SwiperSlide>
+        ))}
+      </Swiper>
+    </CreatorWrap>
+  );
+};
+
+const MessageItem = ({ img, nickname, date, children }) => {
+  return (
+    <Wrap>
+      <div>
+        <img
+          src={
+            img
+              ? img
+              : "https://ssl.pstatic.net/static/pwe/address/img_profile.png"
+          }
+          alt=""
+        />
```

Comment:
```suggestion
        <img
          src={img ? img : DEFAULT_PROFILE}
          alt="user profile"
        />
```
- 문자열을 별도의 상수 변수로 선언해줄 수 있습니다
- alt 속성에 반드시 적절한 문구를 넣어주세요.